specification/2.0/README.md

```diff
@@ -725,7 +727,7 @@ Each attribute is defined as a property of the `attributes` object. The name of
 
 Valid attribute semantic property names include `POSITION`, `NORMAL`, `TANGENT`, `TEXCOORD_0`, `TEXCOORD_1`, `COLOR_0`, `JOINTS_0`, and `WEIGHTS_0`.  Application-specific semantics must start with an underscore, e.g., `_TEMPERATURE`.
 
-Valid accessor type and component type for each attribute semantic property are defined below.
+The valid accessor type and component type for each attribute semantic property is defined below.
```

Comment:
Also, I think I'd prefer "Application-specific semantic property names must..." in the previous paragraph.